### PR TITLE
ci: add yaml files for current triggers

### PR DESCRIPTION
**Pull request by devjgm**
The list of triggers and yaml files will grow, but for now, I'm just
copying what currently exists in the UI.

This PR also adds a script, `trigger.sh`, to make it easier for us to
manipulate these trigger yaml files.

---
This change is [Reviewable](https://reviewable.io/reviews/googleapis/google-cloud-cpp/6092)

### CODE REVIEW

File: ci/cloudbuild/trigger.sh [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+
+# Use getopt to parse and normalize all the args.
+PARSED="$(getopt -a \
+  --options="d:i:lh" \
+  --longoptions="describe:,import:,list,help" \
+  --name="${PROGRAM_NAME}" \
+  -- "$@")"
+eval set -- "${PARSED}"
+
+while true; do
```
Comment (devbww) [inline ci/cloudbuild/trigger.sh:68]: Why a loop?

Comment (devjgm) [reply ci/cloudbuild/trigger.sh:68]: Just left over from when I thought I was going to handle multiple flags at once. But I can add the loop later if it's needed. removed for now.

File: ci/cloudbuild/triggers/clang-tidy.yaml [hunk 2]
```diff
@@ -0,0 +1,20 @@
+---
+createTime: '2021-03-20T22:56:56.896890729Z'
+disabled: true
```
Comment (devbww) [inline ci/cloudbuild/triggers/clang-tidy.yaml:3]: I assume this is still intentional.

**Review by devbww — APPROVED** [review]

File: ci/cloudbuild/triggers/clang-tidy.yaml (hunk 2 above)
Comment (devjgm) [reply ci/cloudbuild/triggers/clang-tidy.yaml:3]: yes it is intentional.

**Pull request merged**